screens/pokedex/pokedex.gui_script

```diff
@@ -153,6 +144,27 @@ local function update_pokedex_info(self)
 	gui.set_text(gui.get_node("txt_caught_kalos"), dex.get_region_seen(dex_data.regions.KALOS))
 end
 
+local function scroll_list_click(self, list)
```

Comment:
Some of these functions had to be shifted up to be above the init function - they used to be called by on_input, but I switched them to be called by init, so they needed to move. I don't think I changed any of them.